san/__init__.py

```diff
@@ -1,3 +1,20 @@
 from .get import get
 from .batch import Batch
 from .api_config import ApiConfig
+import pkg_resources
+import requests
+import json
+from warnings import warn
+
+def get_latest(project):
+    url = 'https://pypi.python.org/pypi/%s/json' % (project)
+    response = requests.get(url).text
+    return json.loads(response)['info']['version']
```

Comment:
May be we can make this check async and if the network call fails, we just ignore it? That should fix the load performance and allow to load the module when there is no network connection (it is a bit useless without it, but still it could be annoying to fail on import I guess)

Reply:
Yeah, added the check when there is no network

Reply:
@valo also added the check to be async